runtime.md

```diff
@@ -34,22 +34,23 @@ This is provided so that consumers can find the container's configuration and ro
 ```
 
 ## Lifecycle
-
-### Create
-
-Creates the container: file system, namespaces, cgroups, capabilities.
-
-### Start (process)
-
-Runs a process in a container.
-Can be invoked several times.
-
-### Stop (process)
-
-Not sure we need that from runc cli.
-Process is killed from the outside.
-
-This event needs to be captured by runc to run onstop event handlers.
+The lifecycle describes the timeline of events that happen from when a container is created to when it ceases to exist.
+
+1. OCI compliant runtime is invoked by passing the bundle path as argument.
```

Comment:
On Tue, Nov 17, 2015 at 05:37:56PM -0800, Vish Kannan wrote:
> > +1. OCI compliant runtime is invoked by passing the bundle path as
> >     argument.
> 
> What is a `runtime`? To me, runtime seems to indicate higher level
> systems.

It's “tools like runC”, see “compliant runtime” in [1] and #107.  What
do you call those?

[1]: https://github.com/opencontainers/specs/blob/v0.1.1/bundle.md